tests/lib/snaps.sh

```diff
@@ -41,9 +41,9 @@ mksnap_fast() {
 
     if [[ "$SPREAD_SYSTEM" == ubuntu-14.04-* ]]; then
         # trusty does not support  -Xcompression-level 1
-        mksquashfs "$dir" "$snap" -comp gzip
+        mksquashfs "$dir" "$snap" -comp gzip -no-fragments
     else
-        mksquashfs "$dir" "$snap" -comp gzip -Xcompression-level 1
+        mksquashfs "$dir" "$snap" -comp gzip -Xcompression-level 1 -no-fragments
```

Comment:
Not blocking on this, but -no-fragments is only interesting for store uploads, and anything using -comp gzip is not going to be uploaded to the store anyway (since the resquash tests would fail).

Reply:
I paused before adding it to that mksquashfs command precisely because of what you mentioned. Even though it isn't going to be uploaded to the store, I think it is still work testing with similar arguments to what all other snaps will be generated with.

Reply:
@chipaca should comment on this part, but iirc gzip is being used to help address travis timeouts because it is faster. Is -no-fragments slower than without it?

Reply:
No, in my testing, the `-no-fragments` option actually speeds up mksquashfs ever so slightly.